### PR TITLE
Included `Sign Up` page and Route fot it.

### PR DESCRIPTION
**Pull request by UserOlena**
Included Sign Up page and created Route for it. The page includes all necessary fields and basic styling to match MVP, but still static. Does not perform any actions at this stage, but is rendered on URL `http://localhost:3000/signup`.

### CODE REVIEW

File: client/src/pages/SignUp.js [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+      marginBottom='10%'
+      {...props}
+    >
+      {'Copyright © '}
+      <Link
+        color='inherit'
+        href='https://github.com/UserOlena/Learnify/blob/main/LICENSE'
+      >
+        Learnify
+      </Link>{' '}
```
Comment (sendusyourbones) [inline client/src/pages/SignUp.js]: Should </Link> and {' '} be on separate lines?

File: client/src/pages/SignUp.js [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+          <Typography component='h1' variant='h5'>
+            Sign up
+          </Typography>
+          <Typography
+            component='h2'
+            variant='h6'
+            textAlign='center'
+            marginTop='1em'
+          >
+            And unlock limitless learning with Lernify's full access
```
Comment (sendusyourbones) [inline client/src/pages/SignUp.js]: Typo: Lernify should be Learnify

File: client/src/pages/SignUp.js (hunk 1 above)
Comment (UserOlena) [reply client/src/pages/SignUp.js]: Fixed.

File: client/src/pages/SignUp.js (hunk 2 above)
Comment (UserOlena) [reply client/src/pages/SignUp.js]: Good catch! Fixed.

**Review by sendusyourbones — APPROVED** [review]
Looks great!

**Pull request merged**